File: BEE 1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var name = lines.shift();
var fixed_salary = parseFloat(lines.shift());
var total_sale = parseFloat(lines.shift());

console.log("TOTAL = R$", (fixed_salary+(total_sale*0.15)).toFixed(2));

File: BEE 1009.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        String name;
        float fixed_salary, total_sale;
        name = input.nextLine();
        fixed_salary = input.nextFloat();
        total_sale = input.nextFloat();
        System.out.printf("TOTAL = R$ %.2f\n", fixed_salary+(total_sale*0.15));
    }
}